frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/RebaseOnParentBackgroundable.java

```diff
@@ -155,4 +164,9 @@ public void run(ProgressIndicator indicator) {
     }
     GitRebaseUtils.rebase(project, Collections.singletonList(gitRepository), params, indicator);
   }
+
+  @Override
+  public void onFinished() {
+    project.getService(SideEffectingActionTrackingService.class).releaseLock();
```

Comment:
`onFinished` executes after `onSuccess`... hence if we show a push dialog in `onSuccess` (as with traverse), then all the actions will still be disabled on rebase while that dialog is showing... not a major issue, but still

Reply:
Maybe try-with-resources for the entire `run`?